README1.md

```diff
@@ -0,0 +1,13 @@
+in order to run the program on linux, instalation of vagrant and virtualbox is requierd installing vagrant:
+
+sudo apt update sudo apt install virtualbox
```

Comment:
Please make sure that commands in README are clear, looks like you forgot to go down a line here, and in line 5 and 9